Source/WebCore/platform/graphics/cg/ImageDecoderCG.h

```diff
@@ -75,6 +75,10 @@ class ImageDecoderCG final : public ImageDecoder {
 
     static String decodeUTI(CGImageSourceRef, const SharedBuffer&);
 
+#if ENABLE(QUICKLOOK_FULLSCREEN)
+    bool shouldUseQuickLookForFullscreen() const;
+#endif
```

Comment:
I think this can be private since you are calling it through `ImageDecoder` not `ImageDecoderCG`.